CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+  - [ ] Prepare the Readme for your new runtime. Make sure to mention the following details
+    - [ ] Docker base image name + version
+    - [ ] HTTP server library name
+    - [ ] Any extra steps needed to get dependencies running (for e.g., `package.json`)
+    - [ ] Copy the rest of the Readme from another existing runtime
+  - [ ] Write the runtime
+    - [ ] Initialize a web server
+      - Use a library that supports `async/await`, also smaller/simpler libraries are preferred as this is a primitive HTTP server
+      - [ ] Set Port 3000
+      - [ ] Bind IP 0.0.0.0
+      - On each POST Request
+        - [ ] Check that the `x-internal-challenge header` matches the `INTERNAL_RUNTIME_KEY` environment variable
+        - [ ] Decode the executor's JSON POST request
+          - [ ] Make sure to have the right default values for Request body fields ([example](https://github.com/open-runtimes/open-runtimes/blob/main/runtimes/node-16.0/server.js#L14-L18))
+           - [ ]  env: empty object
+           - [ ]  headers: empty object
+           - [ ]  payload: empty string
+    - [ ] Create Request Class
+      - [ ] Fields
+        - [ ] env
+        - [ ] payload
+        - [ ] headers
+    - [ ] Create Response Class
+      - [ ] Functions
+        - [ ] send(string data, int statusCode)
+        - [ ] json(object data, int statusCode)
+    - [ ] Execute the function
+      - [ ] Add `try catch` block for error handling
+        - [ ] Check if the file is missing
+        - [ ] Check if the method is missing
+        - [ ] Any other language-specific errors that may occur 
+  - [ ] Write the `build.sh` script
+  - [ ] Write the `start.sh` script
+  - [ ] Write the Dockerfile
+    - [ ] Add the Docker image you want to base your runtime off
+    - [ ] Create the folders you'll use
+    - [ ] Copy your source code and set working directory
+    - [ ] Add execute permissions for any scripts
+      - [ ] `build.sh`
+      - [ ] `start.sh`
+    - [ ] Use `RUN` commands for necessary dependencies (if needed)
+    - [ ] Expose port 3000
+    - [ ] Add a `CMD` command for `start.sh`
+  - [ ] Update `./build.sh` by adding Docker build command for the new runtime
+  - [ ] Add test
+    - [ ] Create a PHP file named by the language   
+    - [ ] Create a new folder in `./tests` by the name of your runtime
+    - [ ] Inside the folder, create a function ([example](https://github.com/open-runtimes/open-runtimes/blob/main/tests/node-16.0/tests.js))
+      - [ ] Decode the payload as JSON
+      - [ ] Set a string variable called `id` to the value of the `id` key in the payload or to `1` if it doesn't exist
+      - [ ] Fetch `https://jsonplaceholder.typicode.com/todos/$id` with an HTTP Client installed from your language's package manager using the `id` variable
```

Comment:
The test must use a external dependency if the runtime supports it, this is to prove that the dependency fetching aspect of the runtime works

Reply:
```suggestion
      - [ ] Fetch `https://jsonplaceholder.typicode.com/todos/$id` with an HTTP Client installed from your language's package manager using the `id` variable
      - The test must use an external dependency (either for the HTTP Client or the JSON serializer) in order to prove that the runtime can fetch dependencies
```

@PineappleIOnic is this okay?

Reply:
Yes, LGTM 👍